lib/public_key/asn1/Makefile

```diff
@@ -18,6 +18,9 @@
 # %CopyrightEnd%
 #
 
+EGEN = ../src
```

Comment:
is the EGEN needed for something? does not seem to be used in current Makefile...

Reply:
it is used in `otp.mk` to know where to find/put generated erlang files.